tutorial/cytoscape/layout_chapter.py

```diff
@@ -134,16 +135,16 @@
                 ('nyc', 'bos')
             )
         ]
-        
+
         elements = nodes + edges
         '''))
     ]),
 
     dcc.Markdown(dedent('''
     ## Display Methods
-    
-    In most cases, the positions of the nodes will not be given. In those 
-    cases, one of the built-in methods can be used. Let's see what happens 
+
+    In most cases, the positions of the nodes will not be given. In those
```

Comment:
```suggestion
    In most cases, the position of the nodes will not be given. In these
```
(unless each node has multiple positions)